web-console/3-project-details/labels.md

```diff
@@ -0,0 +1,99 @@
+# Labels on Detail Pages
+
+## View Labels
+- Labels are currently displayed at the top of most detail pages (Deployments, Pods, Builds, Images, Storage, etc)
+- A new `Labels` section should be added to the bottom of the `Details` tab on all of these pages to allow for a more comprehensive labels area
```

Comment:
@spadgett Can labels be edited for pods or RCs? I would think that this is similar to Environment Variables, where you could see the labels for those things, but they'd only be editable on the DC.